### PR TITLE
Link to concepts from the bond tutorial

### PR DESCRIPTION
**Pull request by markus-da**

### CODE REVIEW

File: docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst [hunk 1]
```diff
@@ -38,29 +41,53 @@ The code for this tutorial can be executed via the ``runFixedRateBond`` function
 Instrument and Holding
 ======================
 
-We start by creating our fixed rate bond instrument. Here are the key terms:
+:ref:`Fixed rate bonds <module-daml-finance-instrument-bond-fixedrate-instrument-67993>`
+pay a constant coupon each coupon period. The coupon is quoted on a yearly basis
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst:45]:
```suggestion
pay a constant coupon on each coupon period. The coupon is quoted on a yearly basis
```

File: docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst [hunk 2]
```diff
@@ -38,29 +41,53 @@ The code for this tutorial can be executed via the ``runFixedRateBond`` function
...
+  defines whether dates are rolled at the end of the month or on a given date of the month. In our
+  example above, we went for the latter option.
+- The :ref:`StubPeriodTypeEnum <type-daml-finance-interface-types-date-schedule-stubperiodtypeenum-69372>`
+  allows you to explicitly specify what kind of stub period the bond should have. This is optional
+  and not used in the example above. Instead, we defined the stub implicitly by specifying a
+  ``firstRegularPeriodStartDate``: since the time between the issue date and the first regular
+  period start date is less than 12M (our regular coupon period), this implies a short initial
+  stub period.
+
+Now that we have defined the terms we can create the bond instrument using a factory:
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst:90]: Just to confirm, the below snippet is called CREATE_FIXED_RATE_BOND_HOLDING_BEGIN. Does it also create the instrument?

Comment (markus-da) [reply docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst:90]: No, the instrument is created earlier, in a separate snippet. Here is a preview on how it looks: https://docs.daml.com/2.7.0/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.html

Comment (matteolimberto-da) [reply docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst:90]: Sorry, I got confused by Github and didn't realize there was some code in between ... 😓 haven't fully woken up yet

Comment (markus-da) [reply docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst:90]: No problem :)
All good to approve?

File: docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst [hunk 3]
```diff
@@ -38,29 +41,53 @@ The code for this tutorial can be executed via the ``runFixedRateBond`` function
 Instrument and Holding
 ======================
 
-We start by creating our fixed rate bond instrument. Here are the key terms:
+:ref:`Fixed rate bonds <module-daml-finance-instrument-bond-fixedrate-instrument-67993>`
+pay a constant coupon rate at the end of each each coupon period. The coupon rate is quoted on an
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/fixed-rate-bond.rst:45]:
```suggestion
pay a constant coupon rate at the end of each coupon period. The coupon rate is quoted on an
```

**Review by matteolimberto-da — APPROVED** [review]
Thanks!

Just one typo left, I approved but disabled auto-merge

**Pull request merged**